docs/_docs/ssl-tls.adoc

```diff
@@ -0,0 +1,181 @@
+= SSL/TLS
+
+This page explains how to configure SSL/TLS encryption between the cluster nodes (server and client) and the clients that connect to your cluster.
+
+== Considerations
+
+All internal connections in the cluster context, as well as cluster's user interaction interfaces, are SSL-enabled. The communication categories are as follows: 
+
+* Between the user and the cluster (node): REST
+* Between the user and the platform clients
+* Between nodes: Network (Messaging, Scalecube)
+
+All SSL configurations activities are performed at the node level.
+
+Apache Ignite does not support direct paths to SSL certificates. Instead, it utilizes PKCS12 and JKS keystore.
+
+== REST
+
+The standard implementation of SSL for REST involves configuring a secure connection on a separate port. Apache Ignite supports HTTP and HTTPS, arch on its own port. 
+
+The Apache Ignite 3.x REST security configuration is as follows:
+
+[source,json]
+----
+"rest": {
+    "dualProtocol": false,
+    "httpToHttpsRedirection": false,
+    "ssl": {
+        "enabled": false,
+        "port": 10400,
+        "portRange": 100,
+        "keyStore": {
+            "type": "PKCS12",
+            "path": "must not be empty",
+            "password": "may be empty"
+        }
+    }
+}
+----
+
+== Clients and JDBC
```

Comment:
There is no example of the tcp client and jdbc configuration. Do you plan to extend the PR or do it in another one? You can check ItSslTest#ClusterWithSsl#clientCanConnectWithSsl and ItSslTest#ClusterWithSsl#jdbcCanConnectWithSsl for an example.

Reply:
Aleksandr, I'm a tech writer, not a developer :) Please provide the example code snippets or the exact links to these code snippets.